ipywidgets/interact.py

```diff
@@ -2,7 +2,13 @@
 import itertools
 import matplotlib.pyplot as plt
 from IPython import get_ipython
+import base64
+import sys
 
+if sys.version_info.major > 2:
+    dict_items = 'items'
+else:
+    dict_items = 'iteritems'
```

Comment:
The `six` package can be used here: https://pythonhosted.org/six/

Reply:
That makes things simpler!

Reply:
Awesome!